### PR TITLE
Fixed webp native memory leak

### PR DESCRIPTION
**Pull request by howardpang**
## Motivation (required)

Fixed bug #2073 

## Test Plan (required)

it is a obvious bug, and i found the native memory do not increase when i use this patch

### CODE REVIEW

**Comment by lambdapioneer** [comment]
Thanks @howardpang ! I'll run this against our internal tests once we have the CLA :)

**Comment by howardpang** [comment]
@facebook-github-bot  i have signed CLA just now

**Comment by lambdapioneer** [comment]
Hi @howardpang, can you try the CLA flow again? If it's successful then the bot will add a tag "CLA Signed" to this PR (like here https://github.com/facebook/fresco/pull/2075).

Let me know when trying again doesn't work and I'll forward it to the team that maintains that flow

**Comment by howardpang** [comment]
@lambdapioneer i have try again, and the result :
 

> There were some errors submitting your CLA:
Email is already associated with an existing contributor.

**Comment by lambdapioneer** [comment]
Hi @howardpang, can you mail cla@fb.com with a link to this PR? They should be able to sort it out quickly :) Seems that the GitHub username that was entered had a typo.

**Comment by howardpang** [comment]
@lambdapioneer OK

File: static-webp/src/main/jni/static-webp/webp.cpp [hunk 1]
```diff
@@ -438,15 +438,12 @@ jobject WebPImage_nativeGetFrame(JNIEnv* pEnv, jobject thiz, jint index) {
...
-    new WebPIterator(),
-    WebPDemuxReleaseIterator
-  };
+  WebPIterator iter;
 
   // Note, in WebP, frame numbers are one-based.
-  if (!WebPDemuxGetFrame(spNativeContext->spDemuxer->get(), index + 1, spIter.get())) {
+  if (!WebPDemuxGetFrame(spNativeContext->spDemuxer->get(), index + 1, &iter)) {
     throwIllegalStateException(pEnv, "unable to get frame");
+    WebPDemuxReleaseIterator(&iter);
```
Comment (lambdapioneer) [inline static-webp/src/main/jni/static-webp/webp.cpp]: We should also call `WebPDemuxReleaseIterator(&iter);` in the `if (!spFrameNativeContext)` branch in lines 451-453

**Pull request closed without merging**